go.mod

```diff
@@ -2,18 +2,18 @@ module k8s.io/test-infra
 
 replace github.com/golang/lint => golang.org/x/lint v0.0.0-20190301231843-5614ed5bae6f
 
-// Pin all k8s.io staging repositories to kubernetes v0.17.3
+// Pin all k8s.io staging repositories to kubernetes v0.18.3
 // When bumping Kubernetes dependencies, you should update each of these lines
 // to point to the same kubernetes v0.KubernetesMinor.KubernetesPatch version
 // before running update-deps.sh.
 replace (
 	cloud.google.com/go => cloud.google.com/go v0.44.3
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.2.0+incompatible
 	golang.org/x/lint => golang.org/x/lint v0.0.0-20190409202823-959b441ac422
-	k8s.io/api => k8s.io/api v0.17.3
-	k8s.io/apimachinery => k8s.io/apimachinery v0.17.3
-	k8s.io/client-go => k8s.io/client-go v0.17.3
-	k8s.io/code-generator => k8s.io/code-generator v0.17.3
+	k8s.io/api => k8s.io/api v0.18.3
```

Comment:
Looks like we can get rid of these replace stanzas?

Reply:
No, why would we?